<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2017-12-05T20:39:18Z" name="Default suite" finished-at="2017-12-05T20:39:20Z" duration-ms="2273">
    <groups>
    </groups>
    <test started-at="2017-12-05T20:39:18Z" name="Default test" finished-at="2017-12-05T20:39:20Z" duration-ms="2273">
      <class name="org.sample.tests.EAWorldLoginPageTest">
        <test-method is-config="true" signature="begin()[pri:0, instance:org.sample.tests.EAWorldLoginPageTest@435ce306]" started-at="2017-12-05T14:39:18Z" name="begin" finished-at="2017-12-05T14:39:19Z" duration-ms="1704" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- begin -->
        <test-method signature="loginTest()[pri:0, instance:org.sample.tests.EAWorldLoginPageTest@435ce306]" started-at="2017-12-05T14:39:19Z" name="loginTest" finished-at="2017-12-05T14:39:19Z" duration-ms="5" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.sample.pages.EAWorldPage.searchButtonclick(EAWorldPage.java:33)
	at org.sample.tests.EAWorldLoginPageTest.loginTest(EAWorldLoginPageTest.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:org.sample.tests.EAWorldLoginPageTest@435ce306]" started-at="2017-12-05T14:39:19Z" name="tearDown" finished-at="2017-12-05T14:39:20Z" duration-ms="522" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginTest status=FAILURE method=EAWorldLoginPageTest.loginTest()[pri:0, instance:org.sample.tests.EAWorldLoginPageTest@435ce306] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.sample.tests.EAWorldLoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
